Makefile

```diff
@@ -1,4 +1,18 @@
-.PHONY: all build titond
+BUILD_DIR=./build/bin
+
+TARGET=$(BUILD_DIR)/titond
+
+.PHONY: all run build titond image image-arm clean
+
+all: run
+
+run: $(TARGET)
+	$(TARGET)
+
+$(BUILD_DIR):
+	mkdir -p $@
+
+$(TARGET): build
 
 build:
 	CGO_ENABLED=0 GOOS=linux go build -o ./build/bin/titond ./cmd/titond/main.go
```

Comment:
Thank you very much Brave! 😍
Can you change `./build/bin/titond` to $TARGET ?

Reply:
Thank you Steven
I updated!